node-module-system/add.js

```diff
@@ -0,0 +1,2 @@
+const add = (first, second) => first + second;
+export default add;
```

Comment:
I see you discovered the downside of exporting arrow functions: you can't export it on one line because export default const is not legal.

This is one reason why I, personally, never export arrow functions. In fact, I find them harder to read so I never declare standalone functions as arrow functions. I prefer using function. But that's my personal preference. I see lots of code that uses arrow functions exclusively.